.github/CODEOWNERS

```diff
@@ -29,22 +29,25 @@
 # directory at the root of the repository and any of its
 # subdirectories.
 #/build/logs/ @doctocat
-/tests/disaster-recovery/regional-dr/ @red-hat-storage/turquoise-squad
+/tests/functional/disaster-recovery/metro-dr/ @red-hat-storage/turquoise-squad
+/tests/functional/disaster-recovery/regional-dr/ @red-hat-storage/turquoise-squad
 /tests/lvmo/ @red-hat-storage/aqua-squad
-/tests/manage/z_cluster/ @red-hat-storage/brown-squad
-/tests/manage/pv_services/ @red-hat-storage/green-squad
-/tests/manage/storageclass/ @red-hat-storage/green-squad
-/tests/manage/monitoring/ @red-hat-storage/blue-squad
-/tests/manage/mcg/ @red-hat-storage/red-squad
-/tests/manage/rgw/ @red-hat-storage/red-squad
-/tests/ecosystem/upgrade/ @red-hat-storage/purple-squad
-/tests/e2e/flowtest/ @red-hat-storage/magenta-squad
-/tests/e2e/lifecycle/ @red-hat-storage/magenta-squad
-/tests/e2e/workloads/ @red-hat-storage/magenta-squad
-/tests/e2e/performance/ @red-hat-storage/grey-squad
-/tests/e2e/scale/ @red-hat-storage/orange-squad
-/tests/encryption @red-hat-storage/green-squad
-/tests/ui/ @red-hat-storage/black-squad
+/tests/functional/z_cluster/ @red-hat-storage/brown-squad
+/tests/functional/pv/pv_services/ @red-hat-storage/green-squad
+/tests/functional/storageclass/ @red-hat-storage/green-squad
+/tests/functional/encryption/ @red-hat-storage/green-squad
+/tests/functional/monitoring/ @red-hat-storage/blue-squad
+/tests/functional/object/mcg/  @red-hat-storage/red-squad
+/tests/functional/object/rgw/  @red-hat-storage/red-squad
+/tests/functional/upgrade/ @red-hat-storage/purple-squad
+/tests/cross_functional/flowtest/ @red-hat-storage/magenta-squad
+/tests/functional/workloads/ @red-hat-storage/magenta-squad
+/tests/cross_functional/system_test/ @red-hat-storage/magenta-squad
+/tests/cross_functional/kcs/ @red-hat-storage/magenta-squad
+/tests/cross_functional/performance/ @red-hat-storage/grey-squad
+/tests/cross_functional/scale/ @red-hat-storage/orange-squad
+/tests/functional/ui/ @red-hat-storage/black-squad
+/tests/cross_functional/ui/ @red-hat-storage/black-squad
 /ocs_ci/ocs/ui/ @red-hat-storage/black-squad
 
```

Comment:
What about these paths?:
tests/disaster-recovery
tests/functional/nfs_feature
tests/functional/odf-cli
tests/functional/pod_and_daemons
tests/functional/workloads
tests/disaster-recovery
tests/cross_functional/flowtest
tests/cross_functional/longevity
tests/cross_functional/resilience

Reply:
Let's get proper squads of those paths, add them and have it merged. @ebenahar can you please help here with squad owners?

Reply:
tests/disaster-recovery -> @red-hat-storage/turquoise-squad
tests/functional/nfs_feature -> @red-hat-storage/brown-squad
tests/functional/odf-cli -> @red-hat-storage/brown-squad
tests/functional/pod_and_daemons -> @red-hat-storage/brown-squad @shivamdurgbuns can we move the test in this path to `z_cluster`?
tests/functional/workloads -> @red-hat-storage/magenta-squad
tests/disaster-recovery -> asked @mashetty330 to move the test in this path to `tests/functional/disaster-recovery?`
tests/cross_functional/flowtest -> @red-hat-storage/magenta-squad
tests/cross_functional/longevity -> @red-hat-storage/magenta-squad
tests/cross_functional/resilience -> TBD. For now let's put it under @red-hat-storage/brown-squad

Reply:
I prefer to have this PR merged as it is when we will incorporate the proposed changes from @ebenahar , so it will be easier to cherry-pick.

If there is some test move needed, the owner of test can as part of the change then change also codeowner file as part of the change. WDYT @ebenahar ?

Reply:
@ebenahar we can move tests/functional/pod_and_daemons to z_cluster as this folder will mostly contain tests that are applicable in common for all ODF

Reply:
> I prefer to have this PR merged as it is when we will incorporate the proposed changes from @ebenahar , so it will be easier to cherry-pick.
> 
> If there is some test move needed, the owner of test can as part of the change then change also codeowner file as part of the change. WDYT @ebenahar ?

fine with me

Reply:
@ebenahar @shyRozen fixed in the following commits:
- turquoise-squad update in https://github.com/red-hat-storage/ocs-ci/pull/9605/commits/c5e3385b561ef9e59a66cf08d03f06e9f71023aa
- magenta-squad update in https://github.com/red-hat-storage/ocs-ci/pull/9605/commits/ae07059b8d372de434023e5cc559a63aed026cf0 
- brown-squad update in https://github.com/red-hat-storage/ocs-ci/pull/9605/commits/e226d2abbb8eac510caa32ed920a71340338eeb7

I also separated the paths by squad-sections for clarity in https://github.com/red-hat-storage/ocs-ci/pull/9605/commits/57f3a7a384f5583a5aad14fb105bf32261884d59 without changing the content or the orders of the paths themselves.